plugins/org.jkiss.dbeaver.ext.oracle.ui/src/org/jkiss/dbeaver/ext/oracle/ui/internal/OracleUIMessages.java

```diff
@@ -139,4 +139,9 @@ private OracleUIMessages() {
     public static String dialog_schema_edit_user_name;
     public static String dialog_schema_edit_user_password;
     public static String dialog_schema_edit_label;
+
+    public static String views_session_manager_viewer_show_background;
+    public static String views_session_manager_viewer_show_background_tasks_tool_tip_text;
```

Comment:
`_tool_tip_text` is a bit verbose prefix. We often use just `_tip`. Same for other occurrences